.github/workflows/python-package-conda.yml

```diff
@@ -62,4 +62,7 @@ jobs:
           source activate genomics.py3
           python -m pytest --ignore test/unit/utils/test_db_access.py test/unit
           python -m pytest test/system
+      - name: Test ugbio_utils with pytest
+        run: |
+          source activate genomics.py3
```

Comment:
conda activate?

Reply:
yes
we run ugbio tests using genomics.py3
This was Ilya's request